src/calendar/CalendarMonth.jsx

```diff
@@ -32,6 +26,12 @@ const CalendarMonth = React.createClass({
     value: CustomPropTypes.momentOrMomentRange,
   },
 
+  componentWillMount() {
+    const lang = moment().localeData();
```

Comment:
Sorry, I think I got lost there. How does `require('moment/locale/${language}')` change the internal state of the calendar and trigger a re-render?

An example of the work needed to make it take a `locale` as a prop is this old PR: https://github.com/onefinestay/react-daterange-picker/pull/56. We use a modified version of those changes for our internal dashboards, which have a language toggle that works without reloading the page (because the changing `locale` prop triggers a re-render of that part of render tree).